INSTALL.md

```diff
@@ -222,6 +222,12 @@ cargo install cargo-deb
 cargo deb --install
 ```
 
+To choose a default terminal app, use Debian's `update-alternatives`.
+
+```sh
+update-alternatives --config x-terminal-emulator
```

Comment:
Is the `update-alternatives` call interactive? So does it just work to copy-paste this? Or is it required to add additional parameters?

Reply:
It's interactive:

```
 ~ update-alternatives --config x-terminal-emulator                                                                                      Sat 17 Nov 2018 02:35:33 PM CET
There are 2 choices for the alternative x-terminal-emulator (providing /usr/bin/x-terminal-emulator).

  Selection    Path                             Priority   Status
------------------------------------------------------------
* 0            /usr/bin/alacritty                50        auto mode
  1            /usr/bin/alacritty                50        manual mode
  2            /usr/bin/gnome-terminal.wrapper   40        manual mode

Press <enter> to keep the current choice[*], or type selection number:

```

Reply:
Awesome, thanks for letting me know!